cocotbext/axi/axis.py

```diff
@@ -95,6 +100,11 @@ def normalize(self):
         else:
             self.tkeep = [1]*n
 
+        if self.tstrb is not None:
+            self.tstrb = self.tstrb[:n] + [self.tstrb[-1]]*(n-len(self.tstrb))
```

Comment:
The open question is if tstrb should be checked here, to validate the validity of the tkeep, tstrb combination.

Reply:
But, I suppose what we could do is roll with the log messages for now and then think about some config options for this sort of thing separately.